plugins/modules/create_blueprint.py

```diff
@@ -108,6 +108,7 @@
         groups: '["users", "wheel"]'
 """
 
+from ansible.module_utils._text import to_native, to_text
```

Comment:
Lint is failing because of these two unused imports: ` to_native, to_text`

Reply:
fixed!